### DISCUSSION TITLE
Turn auto calibration on or off?

### DISCUSSION THREAD

**Discussion by scottbmc** [OrcaSlicer/OrcaSlicer#482] [Q&A]
Just a quick question. I know I have to turn auto calibration off on my Bambu X1C for the flow and pressure advance calibration as described in the Wiki. But do I have to leave it turned off afterwards? I mean for normal prints after the calibration process?

**Comment by kylek29** [comment]
I've done both, if you don't mind the extra time, lost space, and extra filament use, you can leave it on. It doesn't change much once you get a spool dialed in. The calibration should find the best offsets regardless of what is set in the filament preset.

I just wish BambuLab would put the in-memory calibrated values on the device page or in the printer menu, so we could just reference the LIDAR-produced values once and set them in the filament. Save us from having to do the PA test.
